README.md

```diff
@@ -141,6 +141,18 @@ Decide how many nodes you want to run.
 
 TODO
 
+### Exposing elasticsearch service with a route
+
+Obtain the CA cert from Elasticsearch.
+```
+oc extract secret/elasticsearch --to=. --keys=admin-ca
+```
+Create a Secured Route Using Re-encrypt Termination.
+(See also https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html#secured-routes)
+
+In the Re-encrypt termination example, put the contents of the file admin-ca to spec.tls.destinationCACertificate.  
```

Comment:
...example, use the contents of.... for spec.tls....

Reply:
Done.  Thanks, @jcantrill !